.storybook/InputField.js

```diff
@@ -32,84 +32,89 @@ storiesOf('InputField', module)
+              <Input id="form-field-1" defaultValue="Plain text with Icon" />
+              <Icon name="chevronDown" size="12" />
+            </InputField>
           </Box>
         </Flex>
-
         <Flex width={900}>
-          <Box width={1 / 2} pr={3}>
+          <Box width={1 / 2} pl={3}>
             <Text bold pt={4} pb={2}>
-              Unfocused Field
+              With Label and Blue Outline
             </Text>
-            <InputField
-              id="form-field-1"
-              label="First name"
-              defaultValue="Oliver"
-              placeholder="e.g. Oliver"
-            />
+            <InputField color="blue">
+              <Label>Label</Label>
+              <Input id="form-field-3" defaultValue="Typing Text" />
+            </InputField>
           </Box>
           <Box width={1 / 2} pl={3}>
             <Text bold pt={4} pb={2}>
-              Focused Field or Passed Blue
+              With Label, Icon, and Chevron
             </Text>
-            <InputField
-              id="form-field-2"
-              label="Last name"
-              defaultValue="Dumoulin"
-              placeholder="e.g. Dumoulin"
-              color="blue"
-            />
+            <InputField color="blue">
+              <Icon name="user" color="blue" />
+              <Label>Label</Label>
+              <Input id="form-field-3" defaultValue="Typing text with Icon" />
+              <Icon name="chevronDown" size="12" />
+            </InputField>
           </Box>
         </Flex>
 
         <Flex width={900}>
-          <Box width={1 / 2} pr={3}>
+          <Box width={1 / 2} pl={3}>
             <Text bold pt={4} pb={2}>
-              Validation Error
+              Email Address with Form Validation
             </Text>
-            <InputField
-              icon="circleInfo"
-              color="red"
-              info="Please enter a valid email address"
-              label="Email Address"
-              defaultValue="oliver.dumoulin@priceline.c"
-              id="form-field-3"
-              placeholder="example@test.com"
-            />
+            <InputField color="red">
+              <Label>Email address</Label>
+              <Input id="form-field-3" defaultValue="olvier@examp" />
+              <Icon name="warning" color="red" size="20" />
+            </InputField>
           </Box>
           <Box width={1 / 2} pl={3}>
             <Text bold pt={4} pb={2}>
-              Validation Success
+              Email Address with Form Validation 2
             </Text>
-            <InputField
-              icon="checkLight"
-              color="darkGreen"
-              label="Email Address"
-              defaultValue="oliver.dumoulin@priceline.com"
-              id="form-field-3"
-              placeholder="example@test.com"
-            />
+            <InputField color="green">
+              <Label>Email address</Label>
+              <Input id="form-field-3" defaultValue="olvier@example.com" />
```

Comment:
Can we add a placeholder for each one of these? It's just have a better experience with a placeholder!